Assets/Scripts/SelectionManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SelectionManager : MonoBehaviour
```

Comment:
I'm curios why did you choose to select/highlight class to be a separate object than the player? I think I would've used this script on the player object and then raycast from the player forward and if it intersects with something highlight it.

Reply:
I removed the selectionManager script now, but i still can`t seem to figure out why can`t i pick up an item more than once